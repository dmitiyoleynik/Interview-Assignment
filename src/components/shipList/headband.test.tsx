import renderer from 'react-test-renderer';

import HeadBand from 'src/components/shipList/headBand';

test('HeadBand should match snapshot', () => {
  const component = renderer.create(<HeadBand />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
