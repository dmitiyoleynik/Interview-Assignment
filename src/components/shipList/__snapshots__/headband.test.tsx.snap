// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadBand should match snapshot 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
  style={
    Object {
      "width": "100%",
    }
  }
>
  <div
    className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
  >
    <h6
      className="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
      style={
        Object {
          "textAlign": "center",
          "width": "100%",
        }
      }
    >
      List of Ships
    </h6>
  </div>
</header>
`;
